### PR TITLE
downgrade to node 10.x

### PR DESCRIPTION
**Pull request by erendrake**

### CODE REVIEW

File: mdl/package.json [hunk 1]
```diff
@@ -12,6 +12,6 @@
     "phantomjs-prebuilt": "^2.1.16"
   },
   "engines": {
-    "node": ">=0.12.0"
+    "node": "10.x"
```
Comment (gregdasney) [inline mdl/package.json:15]: shouldn't this be "0.10.x"

**Review by gregdasney — APPROVED** [review]

File: mdl/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,15 @@
 FROM iusdev.azurecr.io/web-build:latest AS pythonbuild
+
+# Nodejs
+ENV NODE_VERSION 10.18.0
```
Comment (BertCotton) [inline mdl/Dockerfile:4]: This should probably just be done in the base image: https://github.com/ius-gov/docker-images/blob/master/ius-build-base/Dockerfile

Comment (erendrake) [reply mdl/Dockerfile:4]: thats the issue. i dont want to downgrade everyone, just the mdl image

Comment (BertCotton) [reply mdl/Dockerfile:4]: Oh, gotcha!

**Review by BertCotton — APPROVED** [review]

File: mdl/package.json (hunk 1 above)
Comment (erendrake) [reply mdl/package.json:15]: i dont think we want to allow for an ancient version of node? 0.x is from back in 2017?

**Pull request merged**